cloudbuild.yaml

```diff
@@ -249,6 +253,14 @@ steps:
         cd /workspace/applications/rag/tests
         python3 test_frontend.py "127.0.0.1:8081"
         echo "pass" > /workspace/rag_frontend_result.txt
+
+        # Upload locally stored netflix dataset to GCS bucket mounted as /data
+        gsutil cp ./netflix_titles.csv gs://gke-aieco-rag-$SHORT_SHA-$_BUILD_ID/netflix-shows/netflix_titles.csv
+        ray job submit --working-dir . --address=http://127.0.0.1:8265 -- python ray_job.py
+
+        python3 test_rag.py "http://127.0.0.1:8081/prompt"
```

Comment:
This might flake, in my own tests I noticed the frontend getting OOMKilled after multiple attempts. Will monitor for flakes